Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM composer
```

Comment:
Composer is already included in `webdevops/php-nginx:7.2` and updated on every image build.